app/src/main/res/values/strings.xml

```diff
@@ -1772,5 +1772,7 @@ Alternatively, you can leave a note (with a photo)."</string>
     <string name="label_log_message_contains">Message contains</string>
     <string name="title_logs_filters">Filter messages</string>
     <string name="quest_isAmenityIndoor_outside_covered">Not inside, but covered</string>
+    <string name="quest_barrier_opening_width_gate">What is the width of the opening here</string>
+    <string name="quest_barrier_opening_width_explanation">What is the maximum usable opening width here</string>
```

Comment:
Question mark missing. Also, question and explanation sound redundant. Better put it into the question.

Reply:
Updated as per comment